packages/idempotency/src/PersistenceLayer.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { IdempotencyConfig } from './IdempotencyConfig';
+import { PersistenceLayerInterface } from './PersistenceLayerInterface';
+
+abstract class PersistenceLayer implements PersistenceLayerInterface {
+  public configure(_config: IdempotencyConfig): void {}
```

Comment:
one thing we probably want to do it make this optional. Its optional in Python and it makes sense because those configs arent always needed or wanted

Reply:
If not optional, have a default?

Reply:
![image](https://user-images.githubusercontent.com/111440293/189202177-e5be9063-ef43-42fa-b0dd-8dd4302752c0.png)
I'm not sure if that parameter is optional? it looks like the second parameter is optional

Reply:
https://github.com/awslabs/aws-lambda-powertools-python/blob/160feae22e24708bc5bb4cf979a0c05c99db87a4/aws_lambda_powertools/utilities/idempotency/persistence/base.py#L116

Reply:
removed for now because the majority of these options will not be used by us and those that may be will be hardcoded to some predetermined value